packages/chrono-core/src/chrono.ts

```diff
@@ -59,9 +60,7 @@ export class Chrono<TaskMapping extends TaskMappingBase, DatastoreOptions> exten
     try {
       await promiseWithTimeout(Promise.all(stopPromises), this.exitTimeoutMs);
     } catch (error) {
-      this.emit('stop:failed', { error, timestamp: new Date() });
-    } finally {
-      this.emit('close', { timestamp: new Date() });
+      this.emit(ChronoEvents.FORCIBLY_STOPPED, { error, timestamp: new Date() });
```

Comment:
really didnt see the point in the "close"/"stop" events as they can just add logging after calling chrono.stop

same can be said for the start but 🤷